code_of_conduct.md

```diff
@@ -45,7 +45,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 
 ## 5. Weapons Policy
 
-No weapons will be allowed at Nashville PHP events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately and will only be allowed to return without the weapon. Attendees are further expected to comply with all state and local laws on this matter.
+Except in emergency situations, no unholsterd weapons will be allowed to be openly displayed at Nashville PHP events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen brandishing or using of one of these items in an unholstered or unconcealed manner, will be asked to reholster or conceal said weaponry or in failing that be asked to leave immediately and will only be allowed to return without the weapon. Attendees are further expected to comply with all state and local laws on this matter, including the carrying of required permits, as well as the requirements of the host venue.
```

Comment:
s/unholsterd/unholstered

Perhaps consider "no unconcealed weapons" instead, since pocket knives etc. can be clipped into pockets but are not holstered.

Further, perhaps consider leaving the original language, then add an exception for CCW permit holders. That emphasizes the licensing requirement, and that the weapon is to stay concealed.

Reply:
I agree with @pmjones  here, especially considering the original language with the exception of legal permit holders. 
If I'm carrying concealed, no one should know, I wouldn't want anyone to know. I'm willing to let people assume what they want, if they feel better thinking I'm not carrying, I'm more than willing to let them think that.

Reply:
Agreed with both previous comments.  Something to the effect of, "No weapons are allowed except where permitted by law."

Reply:
Updated.